src/ui/components/notch-style/notch-style.tsx

```diff
@@ -0,0 +1,28 @@
+/* Copyright (c) 2022, VRAI Labs and/or its affiliates. All rights reserved.
+ *
+ * This software is licensed under the Apache License, Version 2.0 (the
+ * "License") as published by the Apache Software Foundation.
+ *
+ * You may not use this file except in compliance with the License. You may
+ * obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { Helmet } from 'react-helmet'
+
+export const NotchStyle = ({ color }: { color: string }) => {
```

Comment:
So I would rename this to be `SafeAreaView` instead of notch style (it isnt just for the notch)

The way I would expect to use it would be 

```jsx
<SafeAreaView background="red">
</SafeAreaView>
```

So refactor the prop name as well

Reply:
its updated. 

If we want to develop more features in `SafeAreaView` in the future, then in we'll need to add some `if` to handle `props` value combination. But I skip this `if` for now.